### PR TITLE
Improve keyboard a11y of time fields in datetime pickers

### PR DESCRIPTION
**Pull request by sushmi21**
Improved the accessibility of DateTime pickers.

Success Criteria:
- The DateTime Pickers should be keyboard accessible

How to test:
- You can use this endpoint to test this PR: https://endpoint-dev-v4.cognigy.ai/599d6418ab91a6f455920996c53347e2a13d660ea1652707ab77bb0ff469a41b
- Test all the Date pickers and DateTime pickers by navigating them using 'TAB' and 'SHIFT +TAB' keys
Previously, you will not be able to navigate out of the time input field (keyboard trap). This should be fixed now

### CODE REVIEW

**Comment by pedily** [comment]
Let's add tests for this as well.
I found an article that describes how we could test keyboard navigation and its outcome with cypress:
https://glebbahmutov.com/blog/test-app-using-the-keyboard/#deleting-the-first-item

**Comment by sushmi21** [comment]
> Let's add tests for this as well. I found an article that describes how we could test keyboard navigation and its outcome with cypress: https://glebbahmutov.com/blog/test-app-using-the-keyboard/#deleting-the-first-item

Very useful library for a11y testing. Thanks for suggesting it. :)

**Comment by sushmi21** [comment]
I am not very sure why test runs fail saying 'Unable to authenticate'.  I don't have issues running the tests locally

File: package-lock.json [hunk 1]
```diff
@@ -4763,6 +4720,15 @@
         "node": ">=12.0.0"
       }
     },
+    "node_modules/cypress-real-events": {
+      "version": "1.7.0",
+      "resolved": "https://cognigy.pkgs.visualstudio.com/_packaging/cognigy-feed/npm/registry/cypress-real-events/-/cypress-real-events-1.7.0.tgz",
```
Comment (pedily) [inline package-lock.json:4725]: Please reinstall this without your `.npmrc` redirecting it to a private repo

Comment (sushmi21) [reply package-lock.json]: Fixed. thanks!

**Comment by pedily** [comment]
@sushmi21 I created a separate request item to handle the "access to the webchat root element" now. It's recurring in many places and it makes our code more vulnerable to inconsistencies.

If a customer has a `<div id="webchatWindow"></div>` on their page, this _breaks_ our webchat.

**Comment by sushmi21** [comment]
> @sushmi21 I created a separate request item to handle the "access to the webchat root element" now. It's recurring in many places and it makes our code more vulnerable to inconsistencies.
> 
> If a customer has another `<div id="webchatWindow"></div>` on their page, this _breaks_ our webchat when we use `document.querySelector`. Instead, having access to a ref of the webchat root element would be safer. `rootElement.querySelector` would take away the risk of "searching in the wrong place"

Since you have created a new ticket, should we just merge this? Or do you want me to handle it in this PR?

**Pull request merged**